src/config/cmd.rs

```diff
@@ -41,6 +41,10 @@ pub struct Args {
     #[arg(long, conflicts_with = "protocol", conflicts_with = "udp")]
```

Comment:
Should also conflict with `icmp` I guess?

Reply:
Same for `udp` above, should conflict with `icmp` as well.

Reply:
Good catch - I added in the missing conflicts.